src/styles/main.scss

```diff
@@ -181,6 +181,6 @@ tr.active {
   }
 
   &.error {
-    background: rgba(253, 0, 0, 0.3);
+    background: red;
```

Comment:
The background color for the error notification is set to 'red', which is a solid color. Consider using a more subtle color or adding transparency to maintain consistency with other notification styles.